mw-webapp/src/logic/wayPage/WayPage.tsx

```diff
@@ -186,6 +143,10 @@ interface WayPageProps {
  */
 export const WayPage = (props: WayPageProps) => {
   const navigate = useNavigate();
+  const currentGoalMetricsVisibility = GoalMetricsWorker.getCurrentGoalMetricsVisibility() === "true" ? true : false;
```

Comment:
Could you return parsed values? This line looks weird

Reply:
Done